frontend/src/components/grid/grid_container.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'; 
+import styled from 'styled-components'; 
+import { GridItem } from './grid_item'; 
+
+const Grid = styled.div`
+    display: grid; 
+    grid: repeat(20, 3rem) / repeat(20, 3rem);  
+    height: auto; 
+    height: 60rem; 
+    width: 60rem; 
+    border: 0.1rem solid #696969;
+`
+
+export const GridContainer = (props) => {
+    let inputs = []; 
+
+    for(let i = 0; i < 20; i++) {
```

Comment:
Can you pull out the grid width & height into an exportable constants? We may use that to set up the initial game state.